scripts/exgfs_atmos_gempak_meta.sh

```diff
@@ -97,11 +126,12 @@ do
   $APRUNCFP $DATA/poescript
   export err=$?; err_chk
 
-  if [ $fhr -eq 126 ] ; then
-    fhr=`expr $fhr + 6`
-  else
-    fhr=`expr $fhr + $fhinc`
-  fi
+      typeset -Z3 fhr
```

Comment:
And now we zero-pad again, but to three places? There should probably be one of these at the very top of the loop, and that's it.

Reply:
Will update in `develop` branch.